src/components/Pacs/pfdcmClient.tsx

```diff
@@ -12,27 +12,28 @@ export interface ImageStatusType {
 
 export interface DataFetchQuery {
   SeriesInstanceUID: string;
-  StudyInstanceUID: string;
```

Comment:
It turns out that specifying `StudyInstanceUID` is necessary for FUJI PACS. I will fix this.